src/index.ts

```diff
@@ -214,6 +238,38 @@ class AddonsSdk {
     });
   };
 
+
+  /**
+   * Initialize the SDK by sending a ready() signal to the Outreach host
+   * and resolving a promise when Outreach host responds with a current user
+   * initialization context
+   *
+   * @returns {Promise<OutreachContext>}
+   * @memberof AddonsSdk
+   */
+  public init = async (): Promise<OutreachContext> => {
+    if (this.initTask) {
+      return this.initTask.promise;
+    }
+
+    this.initTask = new Task<OutreachContext>();
+    this.initTask.promise =  new Promise<OutreachContext>((resolve, reject) => {
+      this.initTask!.onfulfilled = resolve;
+      this.initTask!.onrejected = reject;
+
+      this.ready();
```

Comment:
this will show a warning everytime 
`console.warn("Ready function is depricated. Use instead await sdk.init()");`

Reply:
good catch - solved